docs/releases/current-release-notes.md

```diff
@@ -147,3 +27,10 @@ We've been making many useful updates to the documentation here, especially arou
 * **NEW** [Work Generation](../integration-guides/work-generation.md)
 * [Node Security](../running-a-node/security.md)
 * [Block Confirmation Tracking](../integration-guides/block-confirmation-tracking.md)
+
+---
+
+!!! info "Nano Forum available"
+	The Nano Forum is available at https://forum.nano.org/ as a resource to ask questions and get support when participating on the network. The [Node and Representative Management category](https://forum.nano.org/c/node-and-rep) is a great place to ask node upgrade related questions. 
+
+--8<-- "join-technical-mailing-list.md"
```

Comment:
Was moving this banner to the bottom intentional? Maybe it could be displayed at the top in `node-releases.md`

Reply:
Yeah I felt like it was a bit dominating on the top of the page and with such a small page on most desktop screens you'd still see it when arriving. Going to leave as-is for now and reevaluate placement of these soon.